rviz_common/include/rviz_common/config.hpp

```diff
@@ -133,9 +136,6 @@ class RVIZ_COMMON_PUBLIC Config
   /// The default constructor creates an empty config object.
   Config();
 
-  /// The copy constructor copies only the reference to the data, not the data itself.
-  Config(const Config & source);
-
   /// The converting constructor, makes a Value type Config object with the given value.
```

Comment:
If the copy constructor is not used, perhaps this change would be better (to prevent this from being called).

```suggestion
  Config(const Config &) = delete;
  /// The converting constructor, makes a Value type Config object with the given value.
```

Reply:
(and also remove the `=` operator)